net: lib: tls_credentials: net.tls_credentials.trusted_tfm fails

**Describe the bug**
The test `net.tls_credentials.trusted_tfm` fails.

Observed for:

- `nrf5340dk_nrf5340_cpuapp_ns`
- `nrf9160dk_nrf9160_ns`

**To Reproduce**
Steps to reproduce the behavior:
1. have `nrf9160dk_nrf9160`_ns connected
2. go to your zephyr dir
3. call `./scripts/twister -T tests/net/lib/tls_credentials -p nrf9160dk_nrf9160_ns --device-testing --device-serial /dev/ttyACM0 -v --inline-logs`
4. See console output with error

**Expected behavior**
Valid console output

**Impact**
Not clear

**Logs and console output**
```
/jenkins_workspace/workspace/wister-upstream-zephyr_main_main/zephyr/zephyr/subsys/net/lib/tls_credentials/tls_credentials_trusted.c:12:10: fatal error: psa/protected_storage.h: No such file or directory
   12 | #include <psa/protected_storage.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04.1 LTS
 - Toolchain Zephyr SDK 0.16.1
 - Commit SHA or Version used: zephyr-v3.5.0-1949-g1dc6279d12ea

--- a/subsys/net/lib/tls_credentials/CMakeLists.txt
+++ b/subsys/net/lib/tls_credentials/CMakeLists.txt
@@ -15,3 +15,9 @@ zephyr_library_sources_ifdef(CONFIG_TLS_CREDENTIALS_SHELL
 )
 
 zephyr_library_link_libraries_ifdef(CONFIG_MBEDTLS mbedTLS)
+
+if (CONFIG_TLS_CREDENTIALS_BACKEND_PROTECTED_STORAGE AND CONFIG_BUILD_WITH_TFM)
+  target_include_directories(${ZEPHYR_CURRENT_LIBRARY} PRIVATE
+    $<TARGET_PROPERTY:tfm,TFM_BINARY_DIR>/install/interface/include
+  )
+endif()

